.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -14,6 +14,10 @@ Please use the following link syntaxes:
 
 ### Checklist
 
-- [ ] New tests are added to cover all changes
+<!--
+Please mark your choice with an "x" (eg. [x], not [*])
```

Comment:
Perhaps add a link to https://github.com/blog/1375-task-lists-in-gfm-issues-pulls-comments